src/backends/perf_pt/decode.c

```diff
@@ -191,7 +191,8 @@ perf_pt_next_block(struct pt_block_decoder *decoder, int *decoder_status,
         *addr = 0;
         return true;
     }
-    assert((*decoder_status == 0) || (*decoder_status == pts_ip_suppressed));
+    xassert(*decoder_status == 0 || *decoder_status == pts_ip_suppressed,
```

Comment:
That's checking that we don't get any PT events here, because the earlier call to `handle_events` should have drained them. So that's an internal consistency check.